### DISCUSSION TITLE
Supervised Learning

### DISCUSSION THREAD

**Discussion by mssanamateen** [mssanamateen/MLAlgorithms#8] [Q&A] [answered]
What is Supervised Learning ? share your ideas

**Comment by MohammedAbrarAli** [comment, marked as answer]
Supervised learning is a type of machine learning technique that involves training a model using labelled data. This data consists of inputs and their corresponding desired outputs, allowing the model to learn the relationship between them

**Comment by Mazin7d4** [comment]
Supervised Learning is a process of using trained algorithms and labeled training data to build a model that can recognize and learn patterns in the data and generate accurate predictions for the unknown or unseen data. The process of supervision involves labeling the training data and using the model to generate predictions or classifications for unseen data. A model is considered to be trained once it can produce predictions that meet the predefined accuracy metrics.

**Comment by lavachary** [comment]
Supervised learning is a type of machine learning where the algorithm learns from labeled data, meaning each input to the algorithm is associated with a corresponding target output. The algorithm aims to learn the mapping between the input and the output by generalizing from the labeled data, so that it can predict the output for new, unseen inputs.
